scripts/apps/dashboard/user-activity/components/UserActivityWidget.tsx

```diff
@@ -114,7 +114,10 @@ const GET_GROUPS = (userId, services: any): Array<IGroup> => {
             id: 'created',
             label: gettext('Created by this user'),
             dataSource(fetchFn) {
-                const markedQuery = extensions['markForUser'].extension.exposes.getQueryNotMarkedForAnyoneOrMarkedForMe(userId);
+                const markedQuery = extensions['markForUser']
```

Comment:
what if the extension is not registered, won't it crash with `exposes is not a function of undefined`?

Reply:
Right, forgot the optional chaining